### PR TITLE
add microdata to html

### PR DESCRIPTION
**Pull request by Idrinth**

### CODE REVIEW

File: templates/blog.twig [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+    <meta itemprop="inLanguage" content="{{ lang }}">
+    <meta itemprop="image" content="https://www.bjoern-buettner.me/logo.png">
+    <h1>{{ content.title }}</h1>
+    <p>{{ content.description }}</p>
+    <p>{{ content.costs }}</p>
+    <br/>
+    <ul>
+    {% for post in posts %}
+        <li>
+            <a href="/blog/{{ post.slug }}/{{ lang }}"><h2>{% if lang=='en' %}{{ post.title_en }}{% else %}{{ post.title_de }}{% endif %}</h2></a>
```
Comment (JEM94) [inline templates/blog.twig]: ternary is also possible
`{{ lang is same as('en') ? post.title_en : post.title_de }}`

same for other lang=='en' checks

File: templates/imprint-de.twig [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+    <meta itemprop="name" content="Björn Büttner">
+    <meta itemprop="url" content="https://bjoern-buettner.me">
+    <meta itemprop="logo" content="https://bjoern-buettner.me/logo.png">
+    <meta itemprop="email" content="webmaster@bjoern-buettner.me">
+    <h2>Angaben gemäß § 5 TMG</h2>
+    <p itemprop="address">Björn Büttner<br />
+        Böllerts Höfe 4<br />
+        45479 Mülheim an der Ruhr</p>
+    <h2>Kontakt</h2>
+    <p>Telephone: <span itemprop="telephone">+4917647945826</span><br />
```
Comment (JEM94) [inline templates/imprint-de.twig]: "Telephone" (translation)

File: templates/imprint-en.twig [hunk 3]
```diff
@@ -17,5 +27,5 @@
     It is therefore not possible to guarantee a complete data protection from third party access.</p>
 <p>The use of contact data provided because of the imprint requirement for other purposes, like sending advertisements and information mails, is not acceptable.<br />
     We reserve the right to combat abuse of the contact data legally in cases of abuse, for example spam mails.</p>
-<p>Based on Source: <em><a href="http://www.e-recht24.de/muster-disclaimer.htm" target="_blank" rel="noopener">Disclaimer</a> of eRecht24, the internet law portal of <a href="http://www.e-recht24.de/" target="_blank" rel="noopener">lawyer</a> Sören Siebert.</em></p>
+<p>Based on Source: <em><a href="https://www.e-recht24.de/muster-disclaimer.htm" target="_blank" rel="noopener">Disclaimer</a> of eRecht24, the internet law portal of <a href="http://www.e-recht24.de/" target="_blank" rel="noopener">lawyer</a> Sören Siebert.</em></p>
```
Comment (JEM94) [inline templates/imprint-en.twig:30]: This is missing in german version

**Review by JEM94 — CHANGES_REQUESTED** [review]

File: templates/imprint-en.twig (hunk 3 above)
Comment (Idrinth) [reply templates/imprint-en.twig:30]: it's there but worded slightly differently.

**Review by JEM94 — APPROVED** [review]

**Pull request merged**